packages/material-ui/src/styles/colorManipulator.test.js

```diff
@@ -25,314 +25,310 @@ describe('utils/colorManipulator', () => {
-      assert.strictEqual(
+      expect(
         recomposeColor({
           type: 'hsl',
           values: [100, 50, 25],
         }),
-        'hsl(100, 50%, 25%)',
-      );
+      ).to.equal('hsl(100, 50%, 25%)');
     });
 
     it('converts a decomposed hsla color object to a string` ', () => {
-      assert.strictEqual(
+      expect(
         recomposeColor({
           type: 'hsla',
           values: [100, 50, 25, 0.5],
         }),
-        'hsla(100, 50%, 25%, 0.5)',
-      );
+      ).to.equal('hsla(100, 50%, 25%, 0.5)');
     });
   });
 
   describe('hexToRgb', () => {
     it('converts a short hex color to an rgb color` ', () => {
-      assert.strictEqual(hexToRgb('#9f3'), 'rgb(153, 255, 51)');
+      expect(hexToRgb('#9f3')).to.equal('rgb(153, 255, 51)');
     });
 
     it('converts a long hex color to an rgb color` ', () => {
-      assert.strictEqual(hexToRgb('#a94fd3'), 'rgb(169, 79, 211)');
+      expect(hexToRgb('#a94fd3')).to.equal('rgb(169, 79, 211)');
     });
   });
 
   describe('rgbToHex', () => {
     it('converts an rgb color to a hex color` ', () => {
-      assert.strictEqual(rgbToHex('rgb(169, 79, 211)'), '#a94fd3');
+      expect(rgbToHex('rgb(169, 79, 211)')).to.equal('#a94fd3');
     });
 
     it('idempotent', () => {
-      assert.strictEqual(rgbToHex('#A94FD3'), '#A94FD3');
+      expect(rgbToHex('#A94FD3')).to.equal('#A94FD3');
     });
   });
 
   describe('hslToRgb', () => {
     it('converts an hsl color to an rgb color` ', () => {
-      assert.strictEqual(hslToRgb('hsl(281, 60%, 57%)'), 'rgb(169, 80, 211)');
+      expect(hslToRgb('hsl(281, 60%, 57%)')).to.equal('rgb(169, 80, 211)');
     });
 
     it('converts an hsla color to an rgba color` ', () => {
-      assert.strictEqual(hslToRgb('hsla(281, 60%, 57%, 0.5)'), 'rgba(169, 80, 211, 0.5)');
+      expect(hslToRgb('hsla(281, 60%, 57%, 0.5)')).to.equal('rgba(169, 80, 211, 0.5)');
     });
 
     it('allow to convert values only', () => {
-      assert.deepEqual(hslToRgb(decomposeColor('hsl(281, 60%, 57%)')), 'rgb(169, 80, 211)');
+      expect(hslToRgb(decomposeColor('hsl(281, 60%, 57%)'))).to.equal('rgb(169, 80, 211)');
     });
   });
 
   describe('decomposeColor', () => {
     it('converts an rgb color string to an object with `type` and `value` keys', () => {
       const { type, values } = decomposeColor('rgb(255, 255, 255)');
-      assert.strictEqual(type, 'rgb');
-      assert.deepEqual(values, [255, 255, 255]);
+      expect(type).to.equal('rgb');
+      expect(values).to.deep.equal([255, 255, 255]);
     });
 
     it('converts an rgba color string to an object with `type` and `value` keys', () => {
       const { type, values } = decomposeColor('rgba(255, 255, 255, 0.5)');
-      assert.strictEqual(type, 'rgba');
-      assert.deepEqual(values, [255, 255, 255, 0.5]);
+      expect(type).to.equal(type, 'rgba');
+      expect(values).to.deep.equal(values, [255, 255, 255, 0.5]);
```

Comment:
Isn't this testing against itself?

```suggestion
      expect(values).to.deep.equal([255, 255, 255, 0.5]);
```

There are more instances where this would need to be changed.